### PR TITLE
fixed resource name versions

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

File: service/awsconfig/v4/resource/endpoints/resource.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	// Name is the identifier of the resource.
-	Name = "endpointsv2"
+	Name = "endpointsv4"
```
Comment (xh3b4sd) [inline service/awsconfig/v4/resource/endpoints/resource.go]: This is v4 now and not v2.

File: service/awsconfig/v4/resource/kmskey/resource.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	// Name is the identifier of the resource.
-	Name = "kmskey"
+	Name = "kmskeyv4"
```
Comment (xh3b4sd) [inline service/awsconfig/v4/resource/kmskey/resource.go]: Well, here was not even a version.

File: service/awsconfig/v4/resource/s3object/resource.go [hunk 3]
```diff
@@ -14,7 +14,7 @@ import (
 
 const (
 	// Name is the identifier of the resource.
-	Name = "s3objectv3"
+	Name = "s3objectv4"
```
Comment (xh3b4sd) [inline service/awsconfig/v4/resource/s3object/resource.go]: Not v3.

**Comment by xh3b4sd** [comment]
@kopiczko can this also be handled with your fancy script?

**Comment by xh3b4sd** [comment]
> Would be nice to improve that in scope of copy script somehow. Ideas?

My idea was that you improve your script. :D

**Comment by kopiczko** [comment]
Now the script does the job!

**Pull request merged**